cllc-public-app/ViewModels/ApplicationTypeNames.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+using Newtonsoft.Json.Converters;
+
+namespace Gov.Lclb.Cllb.Public.ViewModels
+{
+    public static class ApplicationTypeNames
+    {
+        public const string TiedHouseExcemption = "Tied House Exemption Application";
```

Comment:
very small typo `TiedHouseEx(c)emption` -> `TiedHouseExemption`

Reply:
good catch, fixed in updated.